bitflow/sample.py

```diff
@@ -23,15 +26,19 @@ def get_metricvalue_by_name(self,metric_name):
 		return m
 
 	def get_timestamp(self):
-		dt = datetime.datetime.strptime(self.timestamp,'%Y-%m-%d %H:%M:%S.%f')
-		return dt.timestamp()
+		ts = np.datetime64(self.timestamp)
+		return ts
+
+	def get_printable_timestamp(self):
+		pts = str(self.timestamp).replace("T"," ")
+		return pts
 
 	def set_timestamp(self,timestamp):
-		self.timestamp = str(timestamp)
+		self.timestamp = np.datetime64(timestamp)
```

Comment:
Is the incomong timestamp supposed to be a string or object? Maybe call it timestamp_str or something, to make it clear?